cli/pcluster/pcluster.py

```diff
@@ -576,6 +576,37 @@ def instances(args):
     if config.parameters.get("Scheduler") == "awsbatch":
         LOGGER.info("Run 'awsbhosts --cluster %s' to list the compute instances", args.cluster_name)
 
+def get_master_server_ip(stack_name, config):
+    ec2 = boto3.client('ec2', region_name=config.region,
+                       aws_access_key_id=config.aws_access_key_id,
+                       aws_secret_access_key=config.aws_secret_access_key)
+
+    master_id = get_master_server_id(stack_name, config)
+
+    try:
+        instance = ec2.describe_instances(InstanceIds=[master_id]) \
+            .get('Reservations')[0] \
+            .get('Instances')[0]
+        ip_address = instance.get('PublicIpAddress')
+        state = instance.get('State').get('Name')
+        if state != 'running' or ip_address is None:
+            LOGGER.info("MasterServer: %s\nCannot get ip address." % state.upper)
+            sys.exit(1)
+        return ip_address
+    except ClientError as e:
+        LOGGER.critical(e.response.get('Error').get('Message'))
+        sys.stdout.flush()
+        sys.exit(1)
+    except KeyboardInterrupt:
+        LOGGER.info('\nExiting...')
+        sys.exit(0)
+
+def get_head_user(parameters, template):
```

Comment:
Could you add the docstring?

Reply:
I removed the `get_head_user` function completely